### PR TITLE
Apply rebranding (#643)

### LINKED ISSUE #643: Apply rebranding
**Describe the feature request**
As a plugin maintainer i want to keep all information about tracetronic up to date

**Upstream changes**
- [ ] change tool references
- [ ] change company references

### PR DESCRIPTION
**Pull request by MartinGroscheTT**
Resolves #643

### CODE REVIEW

File: src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXNewServerStep/help.html [hunk 1]
```diff
@@ -30,33 +30,33 @@
...
-ATX.newServer('TEST-GUIDE', 'ECU-TEST')
+ATX.newServer('test.guide', 'ecu.test')
             </pre>
             <pre>
-ATX.newServer('TEST-GUIDE', 'ECU-TEST', 'http://localhost:8085', false, '', '1')
+ATX.newServer('test.guide', 'ecu.test', 'http://localhost:8085', false, '', '1')
             </pre>
             <pre>
-ATX.newServer atxName: 'TEST-GUIDE', toolName: 'ECU-TEST', fullServerURL: 'https://127.0.0.1:8086/context', uploadToServer: true
+ATX.newServer atxName: 'test.guide', toolName: 'ecu.test', fullServerURL: 'https://127.0.0.1:8086/context', uploadToServer: true
```
Comment (tt-marcel) [inline src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXNewServerStep/help.html:39]: Do we also need the toolName "test.guide"?

Comment (MartinGroscheTT) [reply src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXNewServerStep/help.html:39]: test.guide is not a tool installation but ecu.test (<jenkins url>/configureTools)

File: src/main/resources/de/tracetronic/jenkins/plugins/ecutest/tool/installation/Messages_de.properties [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+ET.IsUnixSystem=Angabe ist nicht erforderlich, da der Jenkins-Master auf einem Unix-System l�uft.\
+    Die Konfiguration ist individuell pro Windows-Agent durchzuf�hren.
+ET.NoInstallation=Die ausgew�hlte ecu.test-Installation ist f�r diesen Knoten nicht konfiguriert.
+ET.NotADirectory={0} ist kein g�ltiges Verzeichnis.
+ET.NotHomeDirectory={0} ist kein g�ltiges ecu.test-Installationsverzeichnis.
+ET.NotRequired=Angabe ist nur erforderlich, falls ecu.test auf dem Jenkins-Master ausgef�hrt werden soll. \
+    Andernfalls ist die Konfiguration individuell pro Windows-Agent durchzuf�hren.
 ETToolProperty.DisplayName=Erweiterte Einstellungen
-ETToolProperty.InvalidProgID={0} ist keine g\u00FCltige ProgID, erlaubt sind ECU-TEST.Application, \
+ETToolProperty.InvalidProgID={0} ist keine g�ltige ProgID, erlaubt sind ECU-TEST.Application, \
```
[the remark this answers is not in the document]
Comment (MartinGroscheTT) [reply src/main/resources/de/tracetronic/jenkins/plugins/ecutest/tool/installation/Messages_de.properties:15]: running the `hpi:run -f pom.xml` task

File: README.md [hunk 3]
```diff
@@ -601,14 +600,14 @@ tool:
...
+    <summary>test.guide server</summary>
 
 ```yml
 unclassified:
     testGuide:
         installations:
-            - name: "TEST-GUIDE"
-              toolName: "ECU-TEST"
+            - name: "test.guide"
+              toolName: "ecu.test"
```
[the remark this answers is not in the document]
Comment (MartinGroscheTT) [reply README.md:610]: [me not](https://github.com/jenkinsci/ecutest-plugin/pull/644#discussion_r1431458819)

File: src/main/java/de/tracetronic/jenkins/plugins/ecutest/util/validation/ATXValidator.java [hunk 4]
```diff
@@ -464,8 +464,8 @@ private Proxy configureProxy(final String proxyUrl) throws MalformedURLException
     }
 
     /**
-     * Parses the TEST-GUIDE server information from JSON response.
-     * Checks for valid license content and minimum supported TEST-GUIDE version.
+     * Parses the test.guide server information from JSON response.
+     * Checks for valid license content and minimum supported test.guide version.
      *
      * @param content     the JSON content
      * @param baseUrl     the base server URL
```
[the remark this answers is not in the document]
Comment (MartinGroscheTT) [reply src/main/java/de/tracetronic/jenkins/plugins/ecutest/util/validation/ATXValidator.java:471]: yes we should but i thought this won't be changed

File: .editorconfig [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2015-2019 TraceTronic GmbH <info@tracetronic.de>
```
[the remark this answers is not in the document]
Comment (MartinGroscheTT) [reply .editorconfig:1]: * header of the `docs\CHANGELOG.md`  
  * this is an old file from the forked repo
  * to keep the history untouched i decided not to change this file
* copyright notice in `.idea\copyright\BSD.xml` 
  * ✅

File: src/main/java/de/tracetronic/jenkins/plugins/ecutest/util/validation/ATXValidator.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (tt-marcel) [reply src/main/java/de/tracetronic/jenkins/plugins/ecutest/util/validation/ATXValidator.java:471]: please add a OR (!license.contains("tracetronic"))

**Review by tt-marcel — APPROVED** [review]

**Pull request merged**